02/app.js

```diff
@@ -1,7 +1,15 @@
 const randomArray = createRandomArray();
 console.log(randomArray);
 
+for(let i=0; i<= randomArray.length; i++) {
+    console.log(randomArray[i]);
+}
```

Comment:
Przy tej pętli pojawia się wartość `undefined`. Dlatego, że iterujesz o jeden element za dużo - powinno być `i < randomArray`. Ponieważ w tablicy jest np. 5 elementów, ale indeksy to 0, 1, 2, 3, 4 czyli wartości o jeden mniej niż elementów w tablicy.